.github/workflows/workflow.yml

```diff
@@ -1,5 +1,5 @@
 name: Build
-on: [push]
+on: [push, pull_request]
```

Comment:
I think this will trigger the action twice when pushing a new commit to an existing pull request

Reply:
See https://github.com/netlify/netlify-cms/commit/8c99ad854d6c3665667ee0d977e66c76973c7325#diff-42c92e55465a3edde413334753e1c2e9

Reply:
Thanks! Fixed.